src/api/app/views/webui/user/show.html.haml

```diff
@@ -0,0 +1,170 @@
+          - content_for :head_javascript do
+            var main_projects = [ #{escape_nested_list(@iprojects)} ];
+            var excl_projects = [];
+          - content_for :ready_function do
+            renderProjectsTable(length=10);
+            $('#excludefilter').change(renderProjectsTable);
+    #iowned.tab
+      - if @owned.blank?
+        %p
+          %i
+            = @displayed_user.login
+            is not involved in any projects
+      - else
+        #iowned_wrapper{ "data-url": url_for(controller: 'package', action: 'show', project: 'REPLACEPRJ', package: 'REPLACEPKG') }
+          - content_for :ready_function do
+            var iowned = [ #{escape_nested_list(@owned)} ];
+            renderPackagesProjectsTable({packages: iowned, length: 15, name: 'iowned_wrapper'});
+.clear
+- if @is_displayed_user
+  .grid_16.alpha.omega.box.box-shadow
+    #reviews
+      .box-header.header-tabs
+        %ul
+          %li
+            %a{href: "#reviews_in", title: "Requests that #{@displayed_user.login} has to review"} Incoming Reviews
+            = sprite_tag('reload', title: 'Reload', class: 'result_reload', data: { table: 'reviews_in_table' })
+            = image_tag('ajax-loader.gif', class: 'result_spinner hidden')
+      #reviews_in.tab
+        = render(partial: 'requests', locals: { id: 'reviews_in_table' })
+  .grid_16.alpha.omega.box.box-shadow
+    #requests
+      .box-header.header-tabs
+        %ul
+          %li
+            %a#requests_in_tab{href: "#requests_in", title: "Requests that #{@displayed_user.login} has to merge"} Incoming Requests
+            = sprite_tag('reload', title: 'Reload', class: 'result_reload', data: { table: 'requests_in_table' })
+            = image_tag('ajax-loader.gif', class: 'result_spinner hidden')
+          %li
+            %a#requests_out_tab{href: "#requests_out", title: "Requests that #{@displayed_user.login} has sent"} Outgoing Requests
+            = sprite_tag('reload', title: 'Reload', class: 'result_reload hidden', data: { table: 'requests_out_table' })
+            = image_tag('ajax-loader.gif', class: 'result_spinner hidden')
+          %li
+            %a#requests_declined_tab{href: "#requests_declined", title: "Requests from #{@displayed_user.login} that are declined"} Declined Requests
+            = sprite_tag('reload', title: 'Reload', class: 'result_reload hidden', data: { table: 'requests_declined_table' })
+            = image_tag('ajax-loader.gif', class: 'result_spinner hidden')
+          %li
+            %a#all_requests_tab{href: "#all_requests", title: "All Requests from #{@displayed_user.login}"} All Requests
+            = sprite_tag('reload', title: 'Reload', class: 'result_reload hidden', data: { table: 'all_requests_table' })
+            = image_tag('ajax-loader.gif', class: 'result_spinner hidden')
+      #requests_in.tab
+        = render(partial: 'requests', locals: { id: 'requests_in_table' })
+      #requests_out.tab
```

Comment:
Class should be before id: https://github.com/brigade/haml-lint/blob/master/lib/haml_lint/linter/README.md#classesbeforeids